src/keyboard.rs

```diff
@@ -1682,7 +1682,7 @@ bitflags! {
     /// Represents the current state of the keyboard modifiers
     ///
     /// Each flag represents a modifier and is set if this modifier is active.
-    #[derive(Default, Debug, Clone, Copy, PartialEq, Eq, Hash)]
+    #[derive(Default, Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
```

Comment:
Problem: `super` is a keyword.

Reply:
And should this also apply to `ModifiersStateSerialize` fields? Presumably... but it will affect output.

Reply:
Ah, yeah.... I guess that's why it was added back then, still don't like that we have `key` suffix.

Reply:
I kind of wish there was such a thing as raw identifiers (opposite of raw keywords)...

I guess either we leave it as-is, or we do something inconsistent in the method name and use a rename for serialization (so that `super` is used in the output).

Reply:
I'd keep it as is probably for now.